interactive_multiselect_printer_test.go

```diff
@@ -0,0 +1,68 @@
+package pterm_test
+
+import (
+	"testing"
+
+	"atomicgo.dev/keyboard"
+	"atomicgo.dev/keyboard/keys"
+	"github.com/MarvinJWendt/testza"
+
+	"github.com/pterm/pterm"
+)
+
+func TestInteractiveMultiselectPrinter_Show(t *testing.T) {
```

Comment:
Thanks for adding tests! We somehow totally missed that for this printer @floaust 😅